lib/smart_answer/calculators/minimum_wage_calculator.rb

```diff
@@ -208,5 +208,13 @@ def charged_accomodation_adjustment(charge, number_of_nights)
     def self.historical_minimum_wage_data
       @historical_minimum_wage_data ||= YAML.load_file(Rails.root.join("lib/data/minimum_wage_data.yml"))[:minimum_wage_data]
     end
+
+    def date_from_environment_variable
```

Comment:
@robinjam, @pmanrubia says he is ready and eager to take over this story (and make the changes) on Monday.